addons/binding/org.openhab.binding.pixometer/src/main/java/org/openhab/binding/pixometer/internal/config/Annotation.java

```diff
@@ -0,0 +1,79 @@
+/**
+ * Copyright (c) 2010-2018 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.pixometer.internal.config;
+
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+
+/**
+ * The {@link Annotation} class is the representing java model for the json result for a annotation from the pixometer
+ * api
+ *
+ * @author Jerome Luckenbach - Initial Contribution
+ *
+ */
+public class Annotation {
+
+    @SerializedName("id")
+    @Expose
```

Comment:
Only if you explicitly do something with expose when creating the builder you need to set the annotation Expose. Other wise it doesn't add anything as all fields are exposed by default.

Reply:
>Other wise it doesn't add anything as all fields are exposed by default.

Sounds for me like i can remove all of them, since they only blow up the code.

Reply:
Yes you can remove them all. In your case it doesn't add any information.